HomeTask1/HomeTask1/Program.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+
+namespace ConsoleApp1
+{
+	class Program
+	{
+		static void Main(string[] args)
+		{
+			Console.ForegroundColor = ConsoleColor.Green;
+			Console.WriteLine("В.В. Маяковский");
+			Console.WriteLine("А вы могли бы");
+			Console.WriteLine(" ");
```

Comment:
Зачем строка с пробелом? Пропуск строки можно сделать, если вывести пустую строку 
```suggestion
			Console.WriteLine("А вы могли бы");
			Console.WriteLine("");
```
или использовать Esc-последовательность `\n`
```suggestion
			Console.WriteLine("А вы могли бы\n");
```